include/openssl/ec.h

```diff
@@ -393,14 +393,21 @@ OPENSSL_EXPORT int EC_GROUP_set_generator(EC_GROUP *group,
 
 
 // EC_POINT_point2bn converts an |EC_POINT| to a |BIGNUM| by serializing the
-// point into the X9.62 form given by |form| then interpretting it as a BIGNUM.
+// point into the X9.62 form given by |form| then interpreting it as a BIGNUM.
 // On success, it returns the BIGNUM pointer supplied or, if |ret| is NULL,
 // allocates and returns a fresh |BIGNUM|. On error, it returns NULL. The |ctx|
 // argument may be used if not NULL.
 OPENSSL_EXPORT OPENSSL_DEPRECATED BIGNUM *EC_POINT_point2bn(
     const EC_GROUP *group, const EC_POINT *point, point_conversion_form_t form,
     BIGNUM *ret, BN_CTX *ctx);
 
+// EC_POINT_bn2point is like |EC_POINT_point2bn|, but converts a |BIGNUM| to an
```

Comment:
this doesn't make sense (nor does the EC_POINT_point2bn documentation :)). An EC point consists of (at least) two coordinates x and y, which are usually of type BIGNUM. So what's the relation between `bn` and `point`?

Reply:
I see what you're saying now, OpenSSL also deprecated these functions for [this reason](https://github.com/openssl/openssl/blob/b76a6c26a254b4cc428275fc0ced56759dd5088a/doc/man7/ossl-guide-migration.pod?plain=1#L1686-L1689).

I've reworded the function documentation to describe the actual relation (there's not really much of one, it's just another conversion of the serialized output).